### PR TITLE
[frontend] Improv Drawer component

### PR DESCRIPTION
**Pull request by RomuDeuxfois**
### Proposed changes

* Improvement of Drawer component (Following [PR 4187](https://github.com/OpenCTI-Platform/opencti/pull/4187))
* Add more elements into the Drawer component like **Fab & SubscriptionAvatar**

Co-autored @Kedae

### CODE REVIEW

File: opencti-platform/opencti-front/src/private/components/common/drawer/Drawer.tsx [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    withPanel: {
+      right: 230,
+    },
+    noPanel: {
+      right: 30,
+    },
+  });
+});
+
+interface DrawerOpenCTIProps {
```
Comment (yassine-ouaamou) [inline opencti-platform/opencti-front/src/private/components/common/drawer/Drawer.tsx]: DrawerProps rather

File: opencti-platform/opencti-front/package.json [hunk 2]
```diff
@@ -20,6 +20,7 @@
     "apexcharts": "3.42.0",
     "axios": "1.5.0",
     "ckeditor5-custom-build": "0.0.1",
+    "classnames": "2.3.2",
```
Comment (SarahBocognano) [inline opencti-platform/opencti-front/package.json:23]: What is the purpose of that lib ?

Comment (Kedae) [reply opencti-platform/opencti-front/package.json:23]: This lib is great to handle complex usages of classname in components.
It allows to have multiple classes or conditionnal classes in a component.

**Review by SarahBocognano — COMMENTED** [review]
Tested ✅ Works fine in general, and also with the banner announcement

**Comment by yassine-ouaamou** [comment]
If we add an announcement banner and refresh the app, it floates instead of being on top of the app zone
banner_issue

**Comment by Kedae** [comment]
> If we add an announcement banner and refresh the app, it floates instead of being on top of the app zone banner_issue

The same behavior is on Testing, can you create a bug for that ?

**Comment by yassine-ouaamou** [comment]
> > If we add an announcement banner and refresh the app, it floates instead of being on top of the app zone banner_issue
> >       
> >        
> The same behavior is on Testing, can you create a bug for that ?

I spoke with Romuald about it and he said he will see if he can fix it in this PR. Otherwise, I will create a bug.

**Comment by RomuDeuxfois** [comment]
> > > If we add an announcement banner and refresh the app, it floates instead of being on top of the app zone banner_issue
> > >       
> > >         [
> > >           
> > >         
> > >             banner_issue
> > >           ](https://user-images.githubusercontent.com/7083405/265743257-77f3637b-5092-46e9-915d-310c45750610.gif)
> > >         
> > >         
> > >           
> > >             
> > >               
> > >             
> > >             
> > >               
> > >               
> > >             
> > >           
> > >           [
> > >             
> > >               
> > >             
> > >           ](https://user-images.githubusercontent.com/7083405/265743257-77f3637b-5092-46e9-915d-310c45750610.gif)
> > 
> > 
> > The same behavior is on Testing, can you create a bug for that ?
> 
> I spoke with Romuald about it and he said he will see if he can fix it in this PR. Otherwise, I will create a bug.

I add this bug here https://github.com/OpenCTI-Platform/opencti/issues/2796, seems to be relativ.

**Comment by RomuDeuxfois** [comment]
PR ready for a second round

**Comment by Kedae** [comment]
Pushed a fix for banner message

**Comment by RomuDeuxfois** [comment]
> Pushed a fix for banner message

Fixed on my side

**Comment by yassine-ouaamou** [comment]
Ok!
I will test it

**Comment by yassine-ouaamou** [comment]
For the annoucement banner, it's working 👍 
For the Policy banner, it is a bit weird because the Drawer is under the top banner but floats under the bottom one
image

**Review by yassine-ouaamou — APPROVED** [review]

**Pull request merged**